tests/store/artifact/test_cloud_artifact_repo.py

```diff
@@ -16,3 +23,67 @@ def test_chunk_size_validation_failure():
         _validate_chunk_size_aws(5 * 1024**2 - 1)
     with pytest.raises(MlflowException, match="Multipart chunk size"):
         _validate_chunk_size_aws(5 * 1024**3 + 1)
+
+@pytest.mark.parametrize("future_result, expected_call_count", [
+    (None, 2),  # Simulate where creds are expired, but successfully download after refresh
+    (Exception("fake_exception"), _MLFLOW_MPD_NUM_RETRIES.get() + 1)  # Simulate where there is a download failure and retries are exhausted
+])
+def test__parallelized_download_from_cloud(monkeypatch, future_result, expected_call_count):
+    print("test__parallelized_download_from_cloud")
+
+    # Mock environment variables
+    monkeypatch.setattr(_MLFLOW_MPD_NUM_RETRIES, 'get', lambda: 3)
+    monkeypatch.setattr(_MLFLOW_MPD_RETRY_INTERVAL_SECONDS, 'get', lambda: 5)
```

Comment:
Can you use `monkeypatch.setenv` instead please?
Can you also set `_MLFLOW_MPD_RETRY_INTERVAL_SECONDS` to 0 so that the tests run quickly.

Reply:
done